src/app/[locale]/_component/selectIdp/idpList.tsx

```diff
@@ -39,7 +40,17 @@ export function IdpList({ spidLevel }: IdpList) {
   };
 
   const getSPID = (IDP: IdentityProvider) => {
+    storageLoginInfoOps.write({
+      idpId: IDP.entityId,
+      idpName: IDP.name,
+      idpSecurityLevel: spidLevel,
+    });
     savePrivilegesData();
+    trackEvent('IO_SESSION_EXIT_LOGIN_IDP_SELECTED', {
+      SPID_IDP_ID: IDP.entityId,
+      SPID_IDP_NAME: IDP.name,
+    });
+    trackEvent('IO_LOGIN_START');
```

Comment:
I know the Figma is not correctly updated. But this event should have the same properties of `IO_LOGIN_SUCCESS`. We're updating the Figma accordingly.

Reply:
ok, let's wait for figma updates

Reply:
Check it out now. It should be updated.

Reply:
@pelliccm what about this?

Reply:
> @pelliccm what about this?

updated @ line 60, 61

Reply:
@pelliccm did you mean lines 50 e 51? The event that must have the properites is `IO_LOGIN_START`:

![immagine](https://github.com/pagopa/io-web-profile/assets/16268789/d9727f63-e0bc-449c-9064-22628c83f92d)

Reply:
Sorry I've seen now. Okay it's good. But some checks are failing.